dipy/denoise/noise_estimate.py

```diff
@@ -58,7 +53,7 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
     sigma : float
         The estimated standard deviation of the gaussian noise.
 
-    mask (optional): ndarray
```

Comment:
I know you didn't write it this way, but why is returning the mask an optional thing in the first place?